embeddings/config/config_space.py

```diff
@@ -6,23 +6,19 @@
 
 import optuna
 
-from embeddings.data.io import T_path
+from embeddings.config.base_config import Config
+from embeddings.config.parameters import ConstantParameter, ParameterValues, SearchableParameter
 from embeddings.embedding.auto_flair import AutoFlairWordEmbedding
 from embeddings.embedding.flair_embedding import FlairTransformerEmbedding
 from embeddings.embedding.static.embedding import StaticEmbedding
-from embeddings.hyperparameter_search.parameters import (
-    ConstantParameter,
-    ParameterValues,
-    SearchableParameter,
-)
 
 Parameter = Union[SearchableParameter, ConstantParameter]
 ParsedParameters = TypeVar("ParsedParameters")
 SampledParameters = Dict[str, Union[ParameterValues, Dict[str, ParameterValues]]]
-ConfigSpace = TypeVar("ConfigSpace", bound="BaseConfigSpace")
+OptimizedConfig = TypeVar("OptimizedConfig", bound="ConfigSpace")
```

Comment:
Any particular reason why using `OptimizedConfig` ? I interpret `OptimizedConfig`  as final config which was obtained from some optimisation process and not the Config Space for performing search

Reply:
It's because BaseConfigSpace was renamed to ConfigSpace when working on the configs. I renamed it back and renamed the TypeVar as well